src/arch/riscv/include/asm/entry.h

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @date 26.11.2019
+ * @author Nastya Nizharadze
+ */
+
+#ifndef RISCV_ENTRY_H_
+#define RISCV_ENTRY_H_
+
+#include <asm/ptrace.h>
+
+#ifndef __ASSEMBLER__
+
+#else /* __ASSEMBLER__ */
+
+	.macro SAVE_ALL
```

Comment:
Can you please provide a link in the comment which described this "mixed" registers order?

Reply:
https://riscv.org/specifications/ page 137

Reply:
x0 | Zero 
x1 | ra 
x2 | sp
x3 | gp 
x4 | tp 
x5 | t0 
x6–7 | t1–2 
x8 | s0/fp 
x9 | s1 
x10–11 | a0–1 
x12–17 | a2–7
x18–27 | s2–11
x28–31 | t3–6